jest.config.js

```diff
@@ -1,22 +1,26 @@
 module.exports = {
   moduleFileExtensions: ["js", "json", "jsx", "node", "ts", "tsx"],
-  preset: "@testing-library/react-native",
+  preset: 'react-native',
+  setupFilesAfterEnv: ['@testing-library/jest-native/extend-expect'],
```

Comment:
This blog post was good: https://keyholesoftware.com/2020/11/02/using-jest-and-testing-library-with-react-native-part-1-setting-up/